spec/ttml1.xml

```diff
@@ -13667,6 +13676,87 @@ and <code>tickRate</code> is the computed value of the <att>ttp:tickRate</att> p
 <p>The above formalisms assumes that the <emph>Root Temporal Extent</emph> corresponds with the beginning of a related media object. If
 this assumption doesn't hold, then an additional offset that accounts for the difference may be introduced when computing media time <code>M</code>.</p>
 </note>
+</div3>
+
+<div3 id="semantics-media-timing">
```

Comment:
The section above here titled "TTML Semantics – Time Expressions and Media Time Relationship" looks like it should belong within this `semantics-media-timing` div3, or possibly be in its own div3. The change here puts it within the **N.2.1 Relationship to Local Real Time** section which that looks strange since it is orthogonal to that. @palemieux would you mind putting it in its own div3?

Reply:
See proposed text.